src/axom/spin/ImplicitGrid.hpp

```diff
@@ -204,8 +204,20 @@ class ImplicitGrid
     for(int i = 0; i < NDIMS; ++i)
     {
       m_bins[i] = BinSet(m_gridRes[i]);
-      m_binData[i] =
-        BinBitMap(&m_bins[i], BitsetType(numElts, allocatorID), 1, allocatorID);
+      m_binData[i] = BinBitMap(&m_bins[i]);
+
+      // Workaround because axom::Array doesn't propagate allocator ID on copy
+      // construction
+      {
+        typename BinBitMap::OrderedMap bitmaps(axom::IndexType {0},
+                                               axom::IndexType {m_gridRes[i]},
+                                               allocatorID);
+        for(int ibin = 0; ibin < m_gridRes[i]; ibin++)
+        {
+          bitmaps.emplace_back(numElts, allocatorID);
```

Comment:
Here's the fix - we emplace our BitSets into a temporary `axom::Array`, which avoids calling the copy constructor. Then we can just move that temporary array over the original bin map array.